ang/civiawards/award-creation/directives/review-panels/review-panels.directive.js

```diff
@@ -556,8 +560,12 @@
         is_active: $scope.currentReviewPanel.isEnabled ? '1' : '0',
         case_type_id: $scope.awardId,
         contact_settings: {
-          exclude_groups: $scope.currentReviewPanel.contactSettings.groups.exclude,
-          include_groups: $scope.currentReviewPanel.contactSettings.groups.include,
+          exclude_groups: $scope.currentReviewPanel.contactSettings.groups.exclude.filter(function (item) {
```

Comment:
Code looks good @lisandro-compucorp , but we need to add unit tests to cover all these scenarios.

Reply:
@deb1990 something like this would be ok? https://github.com/compucorp/uk.co.compucorp.civiawards/pull/223/commits/0ebbd4dca0a50a3cd3e71fb3692649ed40bc34f4